synapse/http/matrixfederationclient.py

```diff
@@ -538,10 +538,10 @@ async def _send_request(
             logger.exception(f"Invalid destination: {request.destination}.")
             raise FederationDeniedError(request.destination)
 
-        if timeout:
-            _sec_timeout = timeout / 1000
-        else:
-            _sec_timeout = self.default_timeout
+        if timeout is None:
+            timeout = int(self.default_timeout)
```

Comment:
So this is also in seconds now...

Reply:
For bonus points, leave a comment saying this is in seconds/milliseconds/whatever.

For a gold star, rename to `self.timeout_seconds` etc!